example/config.yaml

```diff
@@ -0,0 +1,14 @@
+gateways:
+  - gateway_configuration:
+      host_name_suffix: sandbox.platform.hmcts.net
+      ssl_host_name_suffix: sandbox.platform.hmcts.net
+      certificate_name: wildcard-sandbox-platform-hmcts-net
+      exclude_env_in_app_name: true
+      private_ip_address: 10.2.13.122
+    app_configuration:
+      - product: api
+        component: service
+        ssl_enabled: true
+        use_public_ip: true
+        health_path_override: "/status-123"
```

Comment:
on Azure Frontdoor - I have configured a test api with /health/liveness rather than APIM default status. could this be reflective?